src/components/DatePicker/DatePicker.tsx

```diff
@@ -440,13 +440,12 @@ export const DatePicker = chakra(
             <FormField id={`${id}-end-form`}>{endDatePickerElement}</FormField>
           )}
         </DateRangeRow>
-        {helperText && isDateRange && showHelperInvalidText && (
-          <HelperErrorText
-            id={`${id}-helper-text`}
-            isInvalid={false}
-            text={helperText}
-          />
-        )}
+        <HelperErrorText
+          id={`${id}-helper-text`}
+          isInvalid={false}
+          isRenderedText={isDateRange && showHelperInvalidText}
```

Comment:
Why does `isDateRange` matter here?

Reply:
Every `TextInput` has its own helper/error text but there should only be one for the entire component when its in the date range mode.